readme.md

```diff
@@ -78,11 +78,11 @@ interface EncryptedMessageCallback {
 }
 
 interface EncryptedMessage {
-    /** DID key ID of the receiver */
-    receiverKeyId: string
+    /** URI of the key agreement key of the receiver DID used to encrypt the message */
+    receiverKeyUri: string
```

Comment:
This breaking change will go into the next version of the specification, please revert it here